ocaml/sdk-gen/java/main.ml

```diff
@@ -159,36 +159,6 @@ let switch_enum =
 
 let _ = get_java_type switch_enum
 
-(*Helper function for get_marshall_function*)
```

Comment:
I recommend creating an empty file at `ocaml/sdk-gen/java/main.mli` so the ocaml compiler can tell us if there are any unused functions.

Reply:
all done, I've actually added `mli`s where I saw fit. some comments are a bit obvious but I really wanted to have them for the params